content/blog/yaml12-0-1-0/index.Rmd

```diff
@@ -0,0 +1,270 @@
+---
+output: hugodown::hugo_document
+
+slug: yaml12-0-1-0
+title: yaml12 for R and Python
+date: 2025-12-12
+author: Tomasz Kalinowski
+description: >
+    We’re pleased to announce two new YAML 1.2 packages: `yaml12` for R and
+    `py-yaml12` for Python. Both are implemented in Rust and designed for
+    fast, predictable YAML 1.2 parsing, with safe opt-in tag handling and document stream support.
+
+photo:
+  url: https://unsplash.com/photos/green-plant-on-brown-clay-pot-PAEwrnasOvY
+  author: Devin H
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package]
+tags: [yaml, r, python, rust]
+---
+
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()` (optional)
+-->
+
+Today we’re announcing two new packages for parsing and emitting YAML 1.2:
+[`yaml12`](https://posit-dev.github.io/r-yaml12/) for R and
+[`py-yaml12`](https://posit-dev.github.io/py-yaml12/) for Python.
```

Comment:
Now that yaml is in r-lib, I think you probably need to mention the relationship between the packages. Maybe something like: "We have taken oven maintenance from [previous author]" and no reason to switch if you're already successful. This is an experiment providing consistent R and python bindings to a new rust library specifically for yaml 1.2, which as we'll see below, has some particular advantages.